package org.clevercastle.authforge.exception;

import org.clevercastle.authforge.CastleException;

public class UserExistException extends CastleException {
}
